src/components/Navigator/NavigatorCard.vue

```diff
@@ -9,126 +9,111 @@
 -->
 
 <template>
-  <div class="navigator-card" :class="{ 'filter-on-top': renderFilterOnTop }">
-    <div class="navigator-card-full-height">
-      <div class="navigator-card-inner">
-        <div class="head-wrapper">
-          <div class="head-inner">
-            <button
-              aria-label="Close documentation navigator"
-              :id="SIDEBAR_HIDE_BUTTON_ID"
-              class="close-card"
-              :class="{ 'hide-on-large': !allowHiding }"
-              @click="handleHideClick"
```

Comment:
`handleHideClick` this is not removed from the methods, but it was copied to the base card